src/view/render/content.coffee

```diff
@@ -207,6 +204,11 @@ LYT.render.content = do ->
     if prevActive
       prevActive.removeClass "active"
 
+    # We assume that the book structure is <p> -> <span id="#{segment.contentId}">,
+    # so we select the closest p parent to the el.
+    if el?[0].nodeName isnt "p"
```

Comment:
- "el?" is always true, since el is a jQuery element.
- if you're worried el is empty, I would move it inside the next if-statement, where we already check for el.length
- Why not use jQuery's function for checking if it's a p-element? el.is("p")